modules/ROOT/pages/dw-values-functions-update.adoc

```diff
@@ -286,57 +286,11 @@ output application/json
  }
 ----
 
-=== Example
```

Comment:
see if we should restore this example, I assume it should be in the dw source but isn't

Reply:
Yes, this needs to be restored and added to sources.